CryptKeeperZK/crypt-keeper-extension
# feat: request connected identity commitment

## Explanation

This PR adds suppor for revealing connected identity commitment.

Details are below:
- [x] Add request commitment page
- [x] Update demo to get identity commitment
- [x] Add event for revealing commitment
- [x] Update cryptkeeper sdk
- [x] Add tabs permissions
- [x] Fix event format for credentials
- [x] Fix reading connected identity

## More Information

Closes #313
Blocked by #796 

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/799e8fab-3839-45c5-a134-c79a44644839)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/0db1f02c-5ecc-47e7-a283-87f4b128fa45)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/a7037287-807c-4cff-aaeb-c7ac87ba082a)


## Manual Testing Steps

1. Connect to demo
2. Click "Reveal" button and follow the flow
3. Check if identity commitment is displayed on demo
4. Check activity tab
5. Check other websites don't get revealed identity commitment

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
